ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -218,3 +218,14 @@ def iscomplex(x: any):
 @to_ivy_arrays_and_back
 def iscomplexobj(x):
     return ivy.is_complex_dtype(ivy.dtype(x))
+
+
+@to_ivy_arrays_and_back
+def fromfunction(function, shape, *, dtype=float):
```

Comment:
The original function signature also has `**kwargs` that are passed on to the `function`, can you add that here?

Reply:
Sure!